contracts/oracles/ZkETHOracle.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.25;
+
+import { IZkETH } from "../interfaces/IZkETH.sol";
+import { CorrelatedTokenOracle } from "./common/CorrelatedTokenOracle.sol";
+
+/**
+ * @title ZkETHOracle
+ * @author Venus
+ * @notice This oracle fetches the price of zkETH (assuming 1 rzkETH = 1 WETH)
```

Comment:
This assumption is made with the configuration. The contract could be configured not assuming that (providing rzkETH as the underlying token)

Reply:
[76d2721](https://github.com/VenusProtocol/oracle/pull/269/commits/76d2721ab9beaaa821e522abc51b5fbc0eb74097)

Do you want me to redeploy and re-verify the contract to reflect the changes in the docs / constructor parameters?